src/components/step-flow/step-flow.mdx

```diff
@@ -132,6 +132,11 @@ as users who can see the component. Please see below how this has been achieved
 
 <ArgTypes of={StepFlowStories} />
 
+## Translation keys
+
+The following keys are available to override the translations for this component by passing in a custom locale object to the
+[i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
```

Comment:
**suggestion**:  due to the Storybook v7 upgrade, some of the URLs generated are now different so this link would break.

Would you be able to use the following URL @tomdavies73?
```suggestion
[i18nProvider](../?path=/docs/documentation-i18n--docs).
```